### PR TITLE
Fixed coupon redemption report for programs only lists one course id

### PR DESCRIPTION
**Pull request by iivic**
LEARNER-1823

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @iivic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=1397"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

**Comment by iivic** [comment]
screen shot 2017-07-18 at 15 38 49

**Comment by iivic** [comment]
@edx/helio please review.

**Review by mjfrey — APPROVED** [review]

File: ecommerce/extensions/voucher/utils.py [hunk 1]
```diff
@@ -41,6 +41,14 @@
 VoucherApplication = get_model('voucher', 'VoucherApplication')
 
 
+def _add_redemption_course_ids(new_row_to_append, header_row, redemption_course_ids):
+    if set(['Catalog Query', 'Program UUID']).intersection(set(header_row)):
```
Comment (marjev) [inline ecommerce/extensions/voucher/utils.py]: why not `if 'Catalog Query' in header_row or 'Program UUID' in header_row:`?

**Review by marjev — CHANGES_REQUESTED** [review]

**Review by marjev — APPROVED** [review]

**Pull request merged**